### DISCUSSION TITLE
Where to begin troubleshooting if everything seems correct

### DISCUSSION THREAD

**Discussion by rushhee** [plausible/analytics#2724] [Self-Hosted Support] [answered]
This is my current plausible screen:  https://imgur.com/VQQ29EO

Ive set up the proxy through here: https://plausible.io/docs/proxy/guides/cloudflare#:~:text=You%20can%20use%20Cloudflare%20Workers,is%20a%20free%20Cloudflare%20account.

And my requests receive GET 200 success via my website

https://imgur.com/9TCK4ZZ

https://imgur.com/h01j2s2

But it seems I have either configured something incorrectly on my side or ive missed something entirely. Do you guys have any ideas where I can start to troubleshoot?

**Comment by ruslandoga** [comment, marked as answer]
👋 @rushhee 

Judging by your first screenshot, Plausible is running on your local network. I'm not familiar with Cloudflare workers, but I think they still need to be able to access your instance over the public internet to be able to proxy the requests.

**Reply by ruslandoga** [reply]
Might be relevant: https://github.com/plausible/analytics/discussions/2574

**Reply by rushhee** [reply]
Thanks for the quick reply, Huge oversight by me that makes a lot of sense, will report back asap thanks

**Reply by rushhee** [reply]
Once i made plausible public facing and added the url it worked, thanks!
